src/app/features/tasks/task/task.component.scss

```diff
@@ -120,12 +120,18 @@
     border-radius: $task-border-radius;
     // we don't want focus borders for touch only devices
     @include noTouchOnly() {
-       border-color: $task-c-focus;
-       border-width: 1px;
-       border-style: solid !important;
+      border-color: $task-c-focus;
+      border-width: 1px;
+      border-style: solid !important;
     }
   }
 
+  :host:focus.mobile-highlight > .inner-wrapper > & {
+    border-color: $task-c-focus;
+    border-width: 1px;
+    border-style: solid !important;
+  }
+
```

Comment:
This causes mobile task highlighting just after search and will disable after blur.